firehose-writer/src/main/java/com/expedia/www/haystack/pipes/firehoseWriter/FirehoseIsActiveController.java

```diff
@@ -44,9 +44,9 @@
     private final Logger logger;
 
     @Autowired
-    public FirehoseIsActiveController(ProtobufToFirehoseProducer protobufToFirehoseProducer,
-                                      Factory firehoseIsActiveControllerFactory,
-                                      Logger firehoseIsActiveControllerLogger) {
+    FirehoseIsActiveController(ProtobufToFirehoseProducer protobufToFirehoseProducer,
```

Comment:
non `public` class?

Reply:
Yes, no need for it to be public, Spring doesn't care, tests are in the same package.